src/compiler/scala/tools/reflect/FastStringInterpolator.scala

```diff
@@ -33,19 +36,33 @@ trait FastStringInterpolator extends FormatInterpolator {
         try
           parts.mapConserve {
             case lit @ Literal(Constant(stringVal: String)) =>
-              val value =
-                if (isRaw && currentRun.isScala3) stringVal
-                else if (isRaw) {
+              def asRaw = {
+                if (currentRun.isScala3 && !currentRun.isScala3Migration) stringVal
```

Comment:
The original idea was to be noisy only under migration, but generally either deprecation or migration warning, neither error nor silence.